app/views/apps/_configuration_instructions.html.erb

```diff
@@ -0,0 +1,23 @@
+# Add the Airbrake gem to your Gemfile:
+# gem 'airbrake', '~> 13.0'
+
+# Invoke the following command from your terminal:
+# rails g airbrake
+#
+# Then modify the following the generated file
+# config/initializers/errbit.rb with the following flags:
+
+Airbrake.configure do |config|
+  config.host = '<%= errbit_url %>'
+  config.project_id = 1 # required, but any positive integer works
+  config.project_key = '<%= api_key %>'
+
+  # Airbrake supports polling a remote config from airbrake.io,
+  # enabled by default. However, since you are using Errbit and not
+  # Airbrake, this request would fail and generate an XML error
+  # message on stdout.
+  config.remote_config = false
+
+  # The same goes for performance_stats
+  config.performance_stats = false
```

Comment:
what about these two, or are they not needed? https://gist.github.com/2called-chaos/07a62137b75a4bf2261a328bc94ed26f#file-z_airbrake-rb-L32-L33

Reply:
They are certainly fine. I've realized it at the same time you wrote this. Rebased and pushed.